packages/backend/src/sandbox.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs';
+import path from 'path';
+import { NodeVM } from 'vm2';
+import { PluginSandbox, PluginSandboxEnvironment } from './types';
+import { Router } from 'express';
+
+export async function loadPlugin(
+  pluginFileName: string,
+  environment: PluginSandboxEnvironment,
+): Promise<Router> {
+  const sandboxedPath = path.join(__dirname, `./plugins/${pluginFileName}.js`);
+  const sandboxedSource = fs.readFileSync(sandboxedPath).toString('utf8');
+  const vm = new NodeVM({
+    sandbox: {},
+    require: {
+      external: true,
+    },
+  });
+  const sandbox: PluginSandbox = vm.run(sandboxedSource, sandboxedPath).default;
+  return await sandbox(environment);
```

Comment:
the above is pretty lax right now - we may want to tweak and lock down the things you are permitted to require, and maybe we should pass in the logger through the `sandbox` rather than giving it directly to the thing that `vm.run` returns etc. It's a start.